modules/ref-supported-runtimes.adoc

```diff
@@ -0,0 +1,80 @@
+endif::[]
+
+ifndef::upstream[]
+
+. The composite Caikit-TGIS runtime is based on link:https://github.com/opendatahub-io/caikit[Caikit^] and link:https://github.com/IBM/text-generation-inference[Text Generation Inference Server (TGIS)^]. To use this runtime, you must convert your models to Caikit format. For an example, see link:https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/built-tip.md#bootstrap-process[Converting Hugging Face Hub models to Caikit format^] in the link:https://github.com/opendatahub-io/caikit-tgis-serving/tree/main[caikit-tgis-serving^] repository.
+
+. The Caikit Standalone runtime is based on link:https://github.com/caikit/caikit-nlp/tree/main[Caikit NLP^]. To use this runtime, you must convert your models to the Caikit embeddings format. For an example, see link:https://github.com/markstur/caikit-embeddings/blob/df9c9bc93187c0a17cb66b86d609f2cd102be97d/demo/server/bootstrap_model.py[Bootstrap Model^].
+
+. link:https://github.com/IBM/text-generation-inference[Text Generation Inference Server (TGIS)^] is based on an early fork of link:https://github.com/huggingface/text-generation-inference[Hugging Face TGI^]. Red Hat will continue to develop the standalone TGIS runtime to support TGI models. If a model does not work in the current version of {productname-short}, support might be added in a future version. In the meantime, you can also add your own custom runtime to support a TGI model. For more information, see link:{rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
+endif::[]
+
+.Deployment requirements
+
+|===
+| Name | Default protocol | Additonal protocol | Model mesh support | Single node OpenShift support | Deployment mode
+
+| Caikit Text Generation Inference Server (Caikit-TGIS) Serving Runtime for KServe | REST | gRPC | No | Yes | Raw and serverless
+
+| Caikit Standalone ServingRuntime for KServe | REST | gRPC | No | Yes | Raw and serverless 
+
+| OpenVINO Model Server | REST | None | Yes | Yes | Raw and serverless 
+
+| Text Generation Inference Server (TGIS) Standalone Serving Runtime for KServe (3) | gRPC | None | No | Yes | Raw and serverless
+
+| vLLM | REST | None | No | Yes | Raw and serverless 
```

Comment:
Suggest replacing *vLLM* with *vLLM ServingRuntime for KServe*

Reply:
```suggestion
| vLLM ServingRuntime for KServe | REST | None | No | Yes | Raw and serverless 
```